app/views/self_service/advisers/_advisers_table.html.erb

```diff
@@ -11,6 +11,7 @@
         <th class="advisers-table__postcode">
           <%= Adviser.human_attribute_name(:postcode) %>
         </th>
+        <th></th>
```

Comment:
Were you still going to do the column reorder? Or shall I pick it up? 

Don't mind which — though we should have a class to set the width for this `th`. See `/app/stylesheets/components/_firms_table.scss`